pom.client.xml

```diff
@@ -260,14 +260,14 @@
             <linksource>false</linksource>
             <excludePackageNames>
               com.microsoft.azure.template:
-              *.impl.*:
-              *.implementation.*:
+              *.impl*:
+              *.implementation*:
               com.azure.tracing.opencensus*
             </excludePackageNames>
             <groups>
               <group>
                 <title>Azure Core</title>
-                <packages>com.azure.core:com.azure.core.annotations:com.azure.core.credentials:com.azure.core.exception:com.azure.core.http*</packages>
+                <packages>com.azure.core:com.azure.core.annotations:com.azure.core.credentials:com.azure.core.exception:com.azure.core.http*:com.azure.core.configuration:com.azure.core.util</packages>
```

Comment:
Can you also support all packages under the util package please?

Reply:
There's nothing else under `util`. Do you still want me to add `util*` just in case we add something later?

Reply:
Yes, I just proposed moving some packages under util, so doing that eagerly now saves doing it later.

Reply:
Ok sure, will update it.

Reply:
Done